index.js

```diff
@@ -27,6 +27,9 @@ function tokenMiddleware(correct) {
 //set up port
 app.set("port", process.env.PORT || 3002);
 
+//set up listen ip
+app.set("listen-ip", process.env.LISTEN_IP || "localhost");
```

Comment:
If you have a suggestion for a better variable name, i'm up to it

Reply:
BIND_ADDR